### PR TITLE
Fix some test at Processing model 2

### PR DESCRIPTION
**Pull request by rianby64**
I found some missing things at [/html/webappapis/scripting/processing-model-2/](https://github.com/w3c/web-platform-tests/tree/master/html/webappapis/scripting/processing-model-2)

Don't know if this matters or not. But what I saw:
1. cross-origin fails on Firefox and Chrome.
2. this is in Firefox: `window.onerror = function(event, source, lineno, colno, error) { ... };` but in Chrome `window.onerror = function(event[EventError])`

Hope this will be helpful.

### CODE REVIEW

File: html/webappapis/scripting/processing-model-2/compile-error-data-url.html [hunk 1]
```diff
@@ -17,11 +17,11 @@
         ran = true;
         col_value = d;
         assert_equals(typeof a, 'string', 'first arg');
-        assert_equals(b, 'data:text/javascript,for(;) {}', 'second arg');
+        assert_equals(b, 'data:text/javascript,for(;){}', 'second arg');
         assert_equals(typeof c, 'number', 'third arg');
     });
   </script>
-  <script src="data:text/javascript,for(;) {}"></script>
```
Comment (rianby64) [inline html/webappapis/scripting/processing-model-2/compile-error-data-url.html]: Should I delete this space?
If leave it then:
```"data:text/javascript,for(;) {}" !== ""data:text/javascript,for(;)%20{}"```

Comment (zcorpan) [inline html/webappapis/scripting/processing-model-2/compile-error-data-url.html]: Good catch. AFAICT the `{}` will not be percent-escaped when serializing the URL per https://url.spec.whatwg.org/ so just removing the space sounds good.

Comment (zcorpan) [inline html/webappapis/scripting/processing-model-2/compile-error-data-url.html]: (Actually I think the spec says to not percent-encode the space either, but that's not what this test is trying to test.)

**Comment by zcorpan** [comment]
Thank you!

**Comment by jdm** [comment]
https://html.spec.whatwg.org/multipage/webappapis.html#onerroreventhandlernonnull says that the function argument is named `column`...

**Comment by zcorpan** [comment]
But https://html.spec.whatwg.org/multipage/webappapis.html#getting-the-current-value-of-the-event-handler says

> If H is an onerror event handler of a Window object
> Let the function have five arguments, named event, source, lineno, colno, and error.

Since the spec changed from `column` to `colno` (in https://github.com/whatwg/html/commit/9f5fd3e4fb31ea3b168dbeca1624fb604bf7479b), I will assume that the IDL was just forgotten about, and it should really also be `colno`. I can fix that in the spec; that's for catching it. (Let me know if I'm missing something and it really should be `column`.)

**Pull request merged**